### PR TITLE
revise superclass for 'prominent'

### LINKED ISSUE #297: Relax the axiomatisation of prominence
Currently we have this:

image

This causes some [confusion](https://github.com/obophenotype/upheno/issues/510#issuecomment-584346349) as phenotypes with an abnormal prominence of something classify under abnormal position. I vote to simply move the prominence and protruding branches out from under position.

@dosumis @wdahdul @balhoff @sbello

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
closes #297

cc @balhoff 
cc @matentzn

### CODE REVIEW

**Comment by nicolevasilevsky** [comment]
thanks @dosumis, done!

File: src/ontology/pato-edit.obo [hunk 1]
```diff
@@ -9594,7 +9594,6 @@ name: prominent
 def: "A quality inhering in a bearer by virtue of the bearer's standing out or projecting beyond a surface or line." [Dictionary.com:Dictionary.com]
 subset: value_slim
 is_a: PATO:0001598 ! protruding
-is_a: PATO:0015007 ! prominence
```
Comment (matentzn) [inline src/ontology/pato-edit.obo:9597]: why is prominent not anymore a subclass of prominence?

Comment (nicolevasilevsky) [reply src/ontology/pato-edit.obo:9597]: Sorry - I think I misunderstood the request. I'll add it back, but should I move prominence to be a subclass of 'physical quality'?

**Comment by sbello** [comment]
The pull request currently only moves protruding. From the ticket I think we also want to move "prominence" (PATO:0015007) out to be under physical quality.
The change to protruding looks good but the issue in #297 needs both to be moved.

**Comment by nicolevasilevsky** [comment]
@sbello I made prominence a child of 'physical quality'

protruding is also a subclass of 'physical quality'. I think this is a little confusing because we also have a class called prominent that is a child of protruding.

Should these be merged?

**Review by sbello — APPROVED** [review]
looks good

**Pull request merged**